### DISCUSSION TITLE
Missing purchases.products.consume

### DISCUSSION THREAD

**Discussion by Dreaming-Codes** [Byron/google-apis-rs#466] [Q&A]
I'm I missing something or google_androidpublisher3 does not have the following method:
https://developers.google.com/android-publisher/api-ref/rest/v3/purchases.products/consume

**Comment by Byron** [comment]
It's probably in a more recent revision of the API which hasn't been generated yet.
Unfortunately there isn't a reliable way to get notified when that happens.

**Reply by Dreaming-Codes** [reply]
I don't think so, as this is a fundamental API and it's likely not from a recent revision. When will the API be regenerated?

**Reply by Dreaming-Codes** [reply]
@Byron Also is there a way to like call raw endpoints with this library using the already authenticated client or something like that?

**Reply by Byron** [reply]
I recommend using the groupsmigration API as minimal example on how the machinery works. From there it should be quite possible to either alter the code or come up with your own abstractions for raw calls. `yup-oauth2` is the crate for authentication, which is already a great part of the task.
